src/cnd/cnd.ts

```diff
@@ -44,6 +44,56 @@ export interface SwapRequest {
   peer: Peer;
 }
 
+export interface CoreRequestBody {
+  alpha_amount: string;
+  beta_amount: string;
+  alpha_network: string;
+  beta_network: string;
+  alpha_identity: string;
```

Comment:
Hmm, I just realized that this cannot be called `redeem`/`refund` because that is different for Alice and Bob right?

I guess it doesn't really matter, what we are using the address for right?

Reply:
Yes correct, I do not use the `redeem`/`refund` because it means that you would expect different type of identity for Alice and Bob for the same swap. Also, the information  is redundant if you know the role and alpha/beta.

Reply:
I was more thinking from the user's perspective: The question we have to answer is "does the user care, for what the identity is used". If no, then this is fine, if yes we should change it to make it obvious.

The client shouldn't derive things from the URL IMO.